source/_components/google_cloud.markdown

```diff
@@ -0,0 +1,109 @@
+    * [Geocoding](https://console.cloud.google.com/flows/enableapi?apiid=geocoding-backend.googleapis.com)
+6. Set up authentication:
+    1. Visit [this link](https://console.cloud.google.com/apis/credentials/serviceaccountkey)
+    2. From the `Service account` list, select `New service account`.
+    3. In the `Service account name` field, enter any name.
+
+    If you are requsting Text-to-Speech API key:
+
+    4. Don't select a value from the Role list. **No role is required to access this service**.
+    5. Click `Create`. A note appears, warning that this service account has no role.
+    6. Click `Create without role`. A JSON file that contains your `API key` downloads to your computer.
+
+## {% linkable_title Google Cloud Text-to-Speech %}
+
+[Google Cloud Text-to-Speech](https://cloud.google.com/text-to-speech/) converts text into human-like speech in more than 100 voices across 20+ languages and variants. It applies groundbreaking research in speech synthesis (WaveNet) and Google's powerful neural networks to deliver high-fidelity audio. With this easy-to-use API, you can create lifelike interactions with your users that transform customer service, device interaction, and other applications.
+
+### {% linkable_title Pricing %}
+
+The Cloud Text-to-Speech API is priced monthly based on the amount of characters to synthesize into audio sent to the service.
+
+| Feature                       | Monthly free tier         | Paid usage                        |
+|-------------------------------|---------------------------|-----------------------------------|
+| Standard (non-WaveNet) voices | 0 to 4 million characters | $4.00 USD / 1 million characters  |
+| WaveNet voices                | 0 to 1 million characters | $16.00 USD / 1 million characters |
+
+### {% linkable_title Text-to-Speach configuration %}
+
+{% configuration %}
+key_file:
+  description: "The [`API key`](#obtaining-api-key) file to use with Google Cloud Platform. If not specified `os.environ['GOOGLE_APPLICATION_CREDENTIALS']` path will be used."
+  required: false
+  type: string
+language:
+  description: "Default language of the voice, e.g. `en`. Supported languages, genders and voices listed [here](https://cloud.google.com/text-to-speech/docs/voices)."
+  required: false
+  type: string
+  default: "`en`"
+gender:
+  description: "Default gender of the voice, e.g. `Male`. Supported languages, genders and voices listed [here](https://cloud.google.com/text-to-speech/docs/voices)."
+  required: false
+  type: string
+  default: "`Neutral`"
+voice:
+  description: "Default voice name, e.g. `en-US-Wavenet-F`. Supported languages, genders and voices listed [here](https://cloud.google.com/text-to-speech/docs/voices)."
+  required: false
+  type: string
+  default: "(None)"
+encoding:
+  description: "Default audio encoder. Supported encodings are `ogg`, `mp3` and `wav`."
+  required: false
+  type: string
+  default: "ogg"
```

Comment:
https://github.com/home-assistant/home-assistant/pull/23629/files#diff-4153553b826ee202dd6fe96e23bbf232R41

What is the default?

Reply:
```python
DEFAULT_ENCODING = SUPPORTED_ENCODINGS[0]
````